animation_nodes/nodes/object/collection_operations.py

```diff
@@ -0,0 +1,33 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+
+class CollectionOperationsNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_CollectionOperationsNode"
+    bl_label = "Collection Operations"
+
```

Comment:
You can use the `VectorizedSocket.CodeEffect` code effect to perform the vectorization automatically. To know how this works, see the `is_inside_volume.py` node for instance.

Reply:
I'm not sure if this is what you expected, but I tried to implement it in 827cae1.

Reply:
Not exactly. The point of the `VectorizedSocket.CodeEffect` is to let Animation Nodes make us the `executeMultiple` function automatically. So we only have to provide the `executeSingle` function. However, since it is a code effect, the node have to be implemented using the `getExecutionCode` function.

Look at the `is_inside_volume.py` again, there is no `executeMultiple`.

Reply:
Like this: c515eff?